File: test folder 2/old enough.py
age = 42
voter = "no"
if age >= 18 and voter == "yes":
   print("You are able to vote!")
elif age >= 16:
   print("You are old enough to drive ")
elif age >= 15:
   print("You are old enough to have a learners permit")
elif age >= 5:
   print("You are old enough to go to school")
else:
      print("You are too young to go to school")

File: conditionals/old enough.c
#include <stdio.h>

int age = 17;

int main(void){
    if (age >=18){
        printf("You are old enough to vote!\n");
    }else if (age >= 16){
        printf("You are old enought to drive!\n");

    }else if (age == 15){
        printf("You are old enough to get your learners permit!\n");

    }else if (age >= 5){
        printf("You are old enough to go to school!\n");
    }else{
        printf("I'm sorry, you are too young to go to school.\n");
    };
    return 0;
    }
